base/ubi9/entrypoint.sh

```diff
@@ -30,4 +30,25 @@ fi
 
 source kubedock_setup
 
+# Stow
+## Required for https://github.com/eclipse/che/issues/22412
+
+# /home/user/ will be mounted to by a PVC if persistUserHome is enabled
+mountpoint -q /home/user/; HOME_USER_MOUNTED=$?
+
+# This file will be created after stowing, to guard from executing stow everytime the container is started
+STOW_COMPLETE=/home/user/.stow_completed
+
+if [ $HOME_USER_MOUNTED -eq 0 ] && [ ! -f $STOW_COMPLETE ]; then
+    # Create symbolic links from /home/tooling/ -> /home/user/
+    stow . -t /home/user/ -d /home/tooling/ --no-folding -v 2 > /tmp/stow.log 2>&1
```

Comment:
@dkwon17 Does this call to `stow` not do the same thing that the call in the Dockefile does?

Reply:
It does, but only if persistent home is enabled. We run stow in the image build to make links from /home/tooling to /home/user in case persistent home isn't enabled, so that files are still accessible under /home/user as expected

Reply:
Got it.  That makes sense.  I will revert that change and modify the stow ignore file.